app/api/v2/management/api_keys.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+module API::V2
+  module Management
+    class APIKeys < Grape::API
+      resource :api_keys do
+        desc 'Create an api key for service account' do
+          @settings[:scope] = :write_apikeys
+          success API::V2::Entities::APIKey
+        end
+        params do
+          requires :algorithm,
+                   type: String,
+                   allow_blank: false,
+                   desc: 'API key algorithm'
+          requires :uid,
+                   type: String,
+                   allow_blank: false,
+                   desc: 'User UID or Service Account UID'
+          optional :scopes,
+                   type: String,
+                   allow_blank: false,
+                   desc: 'Comma separated scopes'
+        end
+        post do
+          if params[:uid].start_with?(Barong::App.config.uid_prefix)
+            key_holder = User.find_by(uid: params[:uid])
```

Comment:
let's introduce a configuration entry to enable this feature, and we split it in 2:
mgn_api_keys_users
mgn_api_keys_sa
we default them to false so by default this is disabled.